src/styles/index.scss

```diff
@@ -1,3 +1,79 @@
+$hand-transform-origin: bottom center;
+$hand-bottom: 50%;
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: dotted #000 1vmin;
+  border-radius: 50%;
+
+  &__hand-minute {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: absolute;
+    bottom: $hand-bottom;
+    animation: rotate-minute 3600s infinite linear;
+    transform-origin: $hand-transform-origin;
+    animation-timing-function: steps(60);
+  }
+
+  &__hand-second {
+    position: absolute;
+    height: 38vmin;
+    width: 1.5vmin;
+    bottom: $hand-bottom;
+    background-color: #2c8000;
+    animation: rotate-second 60s linear infinite;
+    transform-origin: $hand-transform-origin;
+  }
+
+  &--speed-up {
+    .stopwatch__hand-minute {
+      animation-duration: 600s;
+    }
+    .stopwatch__hand-second {
+      animation-duration: 10s;
+    }
+  }
+
+  &__point {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    height: 5vmin;
+    width: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    transform: translate(-50%, -50%);
+  }
+}
+
+@keyframes rotate-minute {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes rotate-second {
```

Comment:
why do you need same keyframes?
looks like you can create just one keyframe and use in two cases.